backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -57,35 +54,32 @@ public UsersController(WebClientService webClientService, @Value("${config.vanit
     }
 
     @GetMapping("/users/{userId}")
-    public Mono<Object> getUser(@PathVariable String userId) {
+    public ResponseEntity<Object> getUser(@PathVariable String userId) {
         return webClientService.getUser(userId);
     }
 
     @PostMapping("/users")
-    public Mono<ResponseEntity<Object>> createUser(@RequestBody Object body) {
-        Mono<ClientResponse> post = webClientService.createUser(body);
-        return post.flatMap(response -> Mono.just(
-                ResponseEntity.status(response.statusCode())
-                        .headers(getHeaders(response.headers().asHttpHeaders()))
-                        .body(response.bodyToMono(Object.class))));
+    public ResponseEntity<Object> createUser(@RequestBody Object body) {
+        ResponseEntity<Object> post = webClientService.createUser(body);
+        return ResponseEntity.status(post.getStatusCode())
+                .headers(getHeaders(post.getHeaders()))
+                .body(post.getBody());
     }
 
     @PutMapping("/users/{userId}")
-    public Mono<ResponseEntity<Object>> updateUser(@PathVariable String userId, @RequestBody Object body) {
-        Mono<ClientResponse> post = webClientService.updateUser(userId, body);
-        return post.flatMap(response -> Mono.just(
-                ResponseEntity.status(response.statusCode())
-                        .headers(response.headers().asHttpHeaders())
-                        .body(response.bodyToMono(Object.class))));
+    public ResponseEntity<Object> updateUser(@PathVariable String userId, @RequestBody Object body) {
+        ResponseEntity<Object> post = webClientService.updateUser(userId, body);
+        return ResponseEntity.status(post.getStatusCode())
+                .headers(getHeaders(post.getHeaders()))
+                .body(post.getBody());
```

Comment:
I don't think we need to filter the headers of this response. The PUT doesn't return a location header we need to update